crates/bevy_ui/src/layout/debug.rs

```diff
@@ -12,22 +12,31 @@ pub fn print_ui_layout_tree(ui_surface: &UiSurface) {
     let taffy_to_entity: HashMap<NodeId, Entity> = ui_surface
         .entity_to_taffy
         .iter()
-        .map(|(entity, node)| (*node, *entity))
+        .map(|(&ui_entity, &taffy_node)| (taffy_node, ui_entity))
         .collect();
-    for (&entity, roots) in &ui_surface.camera_roots {
-        let mut out = String::new();
-        for root in roots {
+    for (&camera_entity, root_node_set) in ui_surface.camera_root_nodes.iter() {
+        bevy_utils::tracing::info!("Layout tree for camera entity: {camera_entity}");
+        for &root_node_entity in root_node_set.iter() {
+            let Some(implicit_viewport_node) = ui_surface
+                .root_node_data
+                .get(&root_node_entity)
+                .map(|rnd| rnd.implicit_viewport_node)
+            else {
+                continue;
+            };
+            let mut out = String::new();
             print_node(
                 ui_surface,
                 &taffy_to_entity,
-                entity,
-                root.implicit_viewport_node,
+                camera_entity,
+                implicit_viewport_node,
                 false,
                 String::new(),
                 &mut out,
             );
+
+            bevy_utils::tracing::info!("{out}");
```

Comment:
```suggestion
        tracing::info!("Layout tree for camera entity: {camera_entity}");
```
(You will need to update the PR with the latest changes from the base branch, as your branch doesn't import `tracing`)

Reply:
i think you meant
```rust
tracing::info!("{out}");
```
for this one

Reply:
I meant to only copy-paste the note in (parenthesis), but apparently I copy-pasted the whole comment... lol

That one's my bad. Suggestion is fixed now.